.buildkite/pipeline.yml

```diff
@@ -137,6 +148,9 @@ steps:
               - "--app=@build/bb_ipa_url.txt"
               - "--device=IOS_15"
               - "--fail-fast"
+              - "--format=junit"
+              - "--out=reports"
+              - "--format=pretty"
```

Comment:
You could use the new config file support to avoid repeating these options.  

The following could also be filtered out in maze.buildkite.cfg/maze.all.cfg under features/support:
```
--fail-fast
--no-tunnel
--aws-public-ip
```